diff --git a/worker.js b/worker.js
index 8fe26e59cc2ddf7342c5d96b066a3a13a4b4a284..ff5045bbb18a29b1baf0027a62346088386acaf7 100644
--- a/worker.js
+++ b/worker.js
@@ -1,4 +1,4 @@
-import { initialize as initializeWorker } from 'monaco-editor/esm/vs/editor/editor.worker.js';
+import { initialize as initializeWorker } from 'monaco-editor/internal/common/initialize.js';
 /**
  * Create a web worker proxy.
  *
